pyfolio/utils.py

```diff
@@ -626,18 +615,47 @@ def get_symbol_rets(symbol, start=None, end=None):
                                     end=end)
 
 
-def set_legend_location(ax, autofmt_xdate=True):
+def set_legend_location(ax, autofmt_xdate=True, change_colors=False):
```

Comment:
Seems a little odd to do this in a function named `set_legend_location`, could we either rename this, or move the color stuff into a separate function?

Reply:
true, renamed this and updated the docstring